RELEASE_NOTES.md

```diff
@@ -31,3 +31,8 @@ cookbooks to supermarkets.
 The cookbook generator now adds a LICENSE file when creating a new
 cookbook.
 
+## Rename `smoke` tests to `integration` tests.
```

Comment:
@nathenharvey Can you rebase this and shove this into the top of the release notes please